docs/contributors/code/release.md

```diff
@@ -188,16 +188,16 @@ Yes. The method for this is identical to the main Plugin release process. You wi
 
--   The `wp/trunk` branch contains the same version of packages published to npm with the `latest` distribution tag. The WordPress core consumes those packages directly in the `trunk` branch and uses them for public releases.
--   The `wp/next` branch contains the same version of packages published to npm with the `next` distribution tag. Projects should use those packages for development purposes only.
--   A Gutenberg branch targeting a specific WordPress major release (including its further minor increments) is created (example `wp/5.2`) based on the `wp/trunk` Gutenberg branch when the corresponding WordPress release branch is created. (This usually happens when the `beta` or `RC` of the next WordPress major version is released).
+-   The `wp/latest` branch contains the same version of packages as those published to npm with the `latest` distribution tag. The goal here is to have this branch synchronized with the last Gutenberg plugin release, and the only exception would be an unplanned [bugfix release](#standalone-bugfix-package-releases).
+-   The `wp/next` branch contains the same version of packages as those published to npm with the `next` distribution tag. It always gets synchronized with the `trunk` branch. Projects should use those packages for development or testing purposes only.
+-   A Gutenberg branch `wp/*` (example `wp/6.0`) targeting a specific WordPress major release (including its further minor increments) gets created based on the `wp/latest` Gutenberg branch just after the last Gutenberg release planned for inclusion in the next major WordPress release.
 
 Release types and their schedule:
 
--   [Synchronizing Gutenberg Plugin](#synchronizing-gutenberg-plugin) (`latest` dist tag) – publishing happens every two weeks based on the newly created RC1 version of the Gutenberg plugin.
--   [WordPress Releases](#wordpress-releases) (`patch` dist tag) – only when bug or security fixes need to be backported into WordPress Core.
+-   [Synchronizing Gutenberg Plugin](#synchronizing-gutenberg-plugin) (`latest` dist tag) – publishing happens automatically every two weeks based on the newly created `release/*` (example `release/12.8`) branch with the RC1 version of the Gutenberg plugin.
+-   [WordPress Releases](#wordpress-releases) (`patch` dist tag) – publishing gets triggered manually from the `wp/*` (example `wp/6.0`) branch. Once we reach the point in the WordPress major release cycle where we only cherry-pick commits from the Gutenberg repository to the WordPress core, we use `wp/*` branch for npm publishing with the `patch` dist-tag. It's also possible to use older branches to backport bug or security fixes to the corresponding older versions of WordPress Core.
```

Comment:
Might be worth mentioning that this point is usually after the first Beta?

Reply:
Yes, the first beta seems like a special case where we take the content of the latest `wp/latest` into `wp/xx` and do the release. How do we do that? Should we document it.

Reply:
We will have to figure out this part together with @adamziel around WordPress 6.0 beta 1. Maybe it's something that should be scripted and triggered from GitHub UI? Example:
1. Create `wp/6.0` branch from `wp/latest` or maybe `release/13.0` (or whatever the last release is).
2. Request a major version bump in all production packages by inserting an entry in CHANGELOG files with a note that it's the beginning of the future major WP release. That part is confusing even for me to explain. We would have to write something strange that translates to this – this release won't be included in WordPress 6.0 because this is based on the Gutenberg plugin that is planned for WordPress 6.1 😅 

Can we somehow trigger that automatically from GitHub actions based on any event? This feature freeze in Gutenberg is a bit fluid and differs between major WP releases.

Reply:
> Can we somehow trigger that automatically from GitHub actions based on any event? 

I think since the dates for betas are "manual", we need to have a job that we can trigger manually. We can leave this part for later for now, but it's important to document at least 👍

Reply:
I will include some initial notes for now and iterate later based on learning from WordPress 6.0 release process.

Reply:
Updated with https://github.com/WordPress/gutenberg/pull/39389/commits/a0f6fadde9f94b337035c3028b2fd8401b811309.

Reply:
It should be fine to follow more or less the same process that we have until now for creating the release branch, except we do it earlier in the cycle. To be safe, we should create `wp/6.0` straight after we do the last package updates for Beta 1, because that's the point at which the Core feature freeze starts.

> Request a major version bump in all production packages

I know we're not exactly following semver 😅 but even so, isn't it better to keep the major versions for actual breaking changes? Or are there advantages to having them follow the WP release schedule? Considering the block editor can be used independently from WP, it would be good for the versioning logic to also be somewhat independent from it.

Reply:
I guess `minor` would be enough in most of the cases, but I doubt we do a good job including CHANGELOG entries for every new feature or API. We use a lot of experiments as well, so for more high-level packages we probably should do a major version bump every two weeks to have a more honest semver increments. In general, it's hard to have a good semver strategy with so many low-level packages. One thing is clear. We need at least do a minor version bump for every major WP release so we have a way to publish big fixes that target older WP releases.